Source/Dafny/Dafny.atg

```diff
@@ -386,7 +386,7 @@ bool IsSetDisplay() {
   if (la.kind == _lbrace) return true;
   int k = scanner.Peek().kind;
   if (la.kind == _iset && k == _lbrace) return true;
-  if (la.kind == _multiset && (k == _lbrace || k == _openparen)) return true;
+  if (la.kind == _multiset) return true;
```

Comment:
Does this function need to be renamed? Just having the token `"multiset"` isn't enough to know that we're looking at a set display, is it?

Reply:
I had looked at the documentation, and I did not find any other uses of the keyword "multiset" than defining a set display and the multiset type. However, I don't think there is a single case when we could be either parsing an expression or a type, so it's not ambiguous.

Reply:
I'm confused, `multiset<…>` is valid syntax and this function will accept it as a set display, no?
Maybe it would be enough to rename this to `ExprIsSetDisplay`, since we know from the context that we're looking at a set display.

Reply:
Ok, I understand now your renaming suggestion. Let me do it.